exhibitor-standalone/src/main/resources/buildscripts/standalone/gradle/build.gradle

```diff
@@ -1,7 +1,17 @@
+version = '1.0'
```

Comment:
Do you want the standalone to be version 1.0? If you want it to be the current version of exhibitor leave version blank when the fatjar or jar asks for version it will be calculated.